config.lua

```diff
@@ -78,7 +78,31 @@ Config = {
     },
 
     -- Hotwire Settings
-    hotwireChance = 0.5, -- Chance for a successful hotwire
+    hotwireChance = {
+        [0] = 0.5, -- Compacts
```

Comment:
I actually like this change, however instead of fixing the indents like that, could we change to an ENUM style key-value pair. I.E.
```
    [VehicleEnum.Compacts] = 0.5
```
This way we make the code more self-descriptive and steer away from unnecessary comments.

Reply:
I think it is slightly beyond the scope of this PR, but I think this might be a better change to do later down the line, it might involve changing a few resources though.

Reply:
> I think it is slightly beyond the scope of this PR, but I think this might be a better change to do later down the line, it might involve changing a few resources though.

Interesting idea, but in my opinion it wouldn't be much of an improvement. But i am open to every idea if it makes the code cleaner or better for you and everyone.

So something like this?

```lua
VehicleEnum = {
    Compacts = 0,
    Sedans = 0,
}

hotwireChance = {
    [VehicleEnum.Compacts] = 0.5,
    [VehicleEnum.Sedans] = 0.5,
}
```

Reply:
> > I think it is slightly beyond the scope of this PR, but I think this might be a better change to do later down the line, it might involve changing a few resources though.
> 
> Interesting idea, but in my opinion it wouldn't be much of an improvement. But i am open to every idea if it makes the code cleaner or better for you and everyone.
> 
> So something like this?
> 
> ```lua
> VehicleEnum = {
>     Compacts = 0,
>     Sedans = 0,
> }
> 
> hotwireChance = {
>     [VehicleEnum.Compacts] = 0.5,
>     [VehicleEnum.Sedans] = 0.5,
> }
> ```

It's not much of an improvement for this one simple idea, but there are several things that something like this could improve readability/maintainability of all the vehicle resources when used across the board, as well as unifying the indexing method of all vehicle categories.

Something like that would work, but don't forget that the value the enum would have to be a unique integer, otherwise you wouldn't have the desired effect.

Reply:
> > > I think it is slightly beyond the scope of this PR, but I think this might be a better change to do later down the line, it might involve changing a few resources though.
> > 
> > 
> > Interesting idea, but in my opinion it wouldn't be much of an improvement. But i am open to every idea if it makes the code cleaner or better for you and everyone.
> > So something like this?
> > ```lua
> > VehicleEnum = {
> >     Compacts = 0,
> >     Sedans = 0,
> > }
> > 
> > hotwireChance = {
> >     [VehicleEnum.Compacts] = 0.5,
> >     [VehicleEnum.Sedans] = 0.5,
> > }
> > ```
> 
> It's not much of an improvement for this one simple idea, but there are several things that something like this could improve readability/maintainability of all the vehicle resources when used across the board, as well as unifying the indexing method of all vehicle categories.
> 
> Something like that would work, but don't forget that the value the enum would have to be a unique integer, otherwise you wouldn't have the desired effect.

Ah that was a typo. Medical written by Manason had these kind of things as far as I remember. I'll do these changes soon.